docs/core/man/doveadm-save.1.md

```diff
@@ -40,14 +40,19 @@ quota is enforced.
 :   Store mail to specified mailbox instead of INBOX.
 
 **-U** *uid*
-:    Save the mail using the given UID, if possible.
+:   Save the mail using the given UID, if possible.
 
 **-g** *guid*
-:    Save the mail using the given GUID.
+:   Save the mail using the given GUID.
 
 **-r** *received-date*
-:    Save the mail using the given received-date timestamp. This is in the
-     "human timestamp" format as described by [[man,doveadm-search-query,,7]].
+:   Save the mail using the given received-date timestamp. This is in the
+    "human timestamp" format as described by [[man,doveadm-search-query,,7]].
+
+## ARGUMENTS
+
+*mail-file*
+:   The message data to save.
```

Comment:
This is a bit too simplified description. The parameter type is "istream", and since this is the only place using it I guess it could explain further:

 * If it is `-` the message is read from stdin (default)
 * Otherwise it's a path to a file

(With HTTP it's the message contents, but this is man page so it's not really relevant.)

Reply:
How about now?

Reply:
"is resolves"

Reply:
fixed